graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/CqlDirectives.java

```diff
@@ -635,7 +635,7 @@ public class CqlDirectives {
     ALL_AS_STRING =
         new SchemaPrinter(
                 SchemaPrinter.Options.defaultOptions()
-                    .includeDirectives(d -> !defaultDirectives.contains(d.getName()))
+                    .includeDirectives(d -> !defaultDirectives.contains(d))
```

Comment:
This also looks strange, was this an error before?

Reply:
Yeah it's a consequence of https://github.com/graphql-java/graphql-java/pull/2562. To be fair that PR mostly preserves backward compatibility, you can still use the incorrect `GraphQLDirective` in most places. But this is one exception where it's not possible (because the erasure of the two lambdas would be the same I think).

+1 to your change, however I think we should also migrate to `GraphQLAppliedDirective` where applicable (the old API still works, but better use the right one). I'll take a look.

Reply:
Actually it turns out that we mostly parse directive applications via the SDL API: `graphql.language.Directive`.

We have a few references to `GraphQLDirective`, but they relate to directive definitions, so they don't need to change.